packages/tasks/src/model-libraries.ts

```diff
@@ -150,6 +150,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"adapter_config.json"`,
 	},
+	comfy: {
```

Comment:
```suggestion
	comfyui: {
```
(nit)
(or `comfy-ui`?)

Reply:
Good catch - I think it should be `comfyui`, at least that's how they use their name shows up on GH: https://github.com/comfyanonymous/ComfyUI